### ISSUE TITLE
[Question] Group items together

### ISSUE THREAD

**Issue by sploders101** [haltu/muuri#417]
Is there a way to group multiple Muuri items together and have them move accordingly?

Use case:
I would like to use this for programming stage lights in my application. A DMX universe is 512 channels, with lights using any number of them. I would like to be able to drag lights onto a grid of 512 squares as a method of assigning IDs to them, to increase visibility and understanding of their usage of a limited address space. Is this possible with Muuri or will I have to develop my own solution?

Thanks!

**Comment by niklasramo** [comment]
@sploders101 Do you mean like dragging multiple items as a group? If so then that kind of feature is not yet built into Muuri, but I'm currently exploring how change the API to make that kind of stuff easy to do.

**Comment by sploders101** [comment]
Yes, that is what I was referring to. Each light can use a variable number of channels. I'm letting users drag the lights onto a grid of 512 blocks, and the channel blocks within the fixture need to stay together, since that's how they're configured. The address you plug into a light is an offset, and they use x number of channels from that offset.

It's not a high priority for me, because though it's not nearly as fluid as muuri, I was able to make a solution for it. It just doesn't show up until you hover over a block, at which point the block changes its information, rather than just being a slot to drop it into.

However, I do think it would be nice to be able to *handle* those edge cases, even if you don't implement it yourself.

Thanks!

**Comment by niklasramo** [comment]
I created a feature issue for this one: https://github.com/haltu/muuri/issues/418. If you have any input for that please do share 🙂
